src/main/java/net/fabricmc/installer/server/ServerInstaller.java

```diff
@@ -61,13 +62,35 @@ public class ServerInstaller {
 	public static final String DEFAULT_LAUNCH_JAR_NAME = "fabric-server-launch.jar";
 	private static final Pattern SIGNATURE_FILE_PATTERN = Pattern.compile("META-INF/[^/]+\\.(SF|DSA|RSA|EC)");
 
-	public static void install(Path dir, LoaderVersion loaderVersion, String gameVersion, InstallerProgress progress) throws IOException {
+	public static void install(Path dir
+			, LoaderVersion loaderVersion
+			, String gameVersion
+			, InstallerProgress progress) throws IOException {
+
+		Scanner sc = new Scanner(System.in);
```

Comment:
No, this will basically break when installing via the GUI. It should show a warning message box. Take a look at the client one for when the launcher is already open.

Reply:
No problem, I will improve the code according to what is needed.